js/packages/teams-ai/src/internals/Colorize.spec.ts

```diff
@@ -0,0 +1,73 @@
+import assert from 'assert';
+import { Colorize } from './Colorize';
+
+describe('Colorize', () => {
+    describe('error', () => {
+        it('should return the error message in red color', () => {
+            const error = new Error('Test error');
+            const result = Colorize.error(error);
+            assert.equal(result, '\x1b[31;1mTest error\x1b[0m');
+        });
+
+        it('should return the string error in red color', () => {
+            const error = 'Test error';
+            const result = Colorize.error(error);
+            assert.equal(result, '\x1b[31;1mTest error\x1b[0m');
+        });
+    });
+
+    describe('output', () => {
+        it('should return the string output in green color', () => {
+            const output = 'Test output';
+            const result = Colorize.output(output);
+            assert.equal(result, '\x1b[32mTest output\x1b[0m');
+        });
+
+        // Does not pass on remote pipeline
```

Comment:
Do we want to remove this code?

Reply:
yep, sorry about that. I was still emotionally attached haha.